packages/mobile/src/import/saga.ts

```diff
@@ -73,21 +82,28 @@ export function* importBackupPhraseSaga({ phrase, useEmptyWallet }: ImportBackup
           Logger.info(TAG + '@importBackupPhraseSaga', 'Using suggested mnemonic autocorrection')
           mnemonic = correctedPhrase
           checkedBalance = true
+        } else {
```

Comment:
nit: There an `if (timeout)` above that logs something. We can probably remove that and move the log to this `else` block

Reply:
Resolved in 87f690d